ui/summary_test.go

```diff
@@ -164,3 +165,26 @@ func createTestMetrics() map[string]*stats.Metric {
 
 	return metrics
 }
+
+func TestSummarizeMetricsJSON(t *testing.T) {
+	metrics := createTestMetrics()
+	expected := `{\n    "name": "",\n    "path": "",\n    "id": "d41d8cd98f00b204e9800998ecf8427e",\n    "groups": {\n        "child": {\n            "name": "child",\n            "path": "::child",\n            "id": "f41cbb53a398ec1c9fb3d33e20c9b040",\n            "groups": {},\n            "checks": {\n                "check1": {\n                    "name": "check1",\n                    "path": "::child::check1",\n                    "id": "6289a7a06253a1c3f6137dfb25695563",\n                    "passes": 5,\n                    "fails": 10\n                }\n            }\n        }\n    },\n    "checks": {}\n}\n{\n    "checks": {\n        "extra": [\n            "✓ 3",\n            "✗ 0"\n        ],\n        "value": 0\n    },\n    "http_reqs": {\n        "count": 3,\n        "rate": 3\n    },\n    "my_trend": {\n        "avg": 15,\n        "max": 20,\n        "med": 15,\n        "min": 10,\n        "p(90)": 19,\n        "p(95)": 19.5\n    },\n    "vus": {\n        "extra": [\n            "min=1",\n            "max=1"\n        ],\n        "value": 1\n    }\n}\n`
```

Comment:
Can you write this in a readable way using multiline string ?

Reply:
Fixed